### ISSUE TITLE
Directions for connecting with SASL for thelounge IRC client

### ISSUE THREAD

**Issue by samebchase** [Libera-Chat/libera-chat.github.io#225]
I have been a thelounge IRC user and have been connecting to Libera.Chat for several months now. As of today, it seems I am locked out of Libera.Chat, and the message I am getting:

```
*** Notice -- SASL authentication to a NickServ account with a verified email address is required to connect from your current network. Please see https://libera.chat/guides/sasl for configuration assistance.
13:00
*** Closing Link: (SASL authentication to a NickServ account with a verified email address is required to connect from your current network. Please see https://libera.chat/guides/sasl for configuration assistance.) (irc)
```

Would be amazing if there are some directions given for [TheLounge](https://thelounge.chat/) client as well. I'd be happy to contribute these instructions to the docs, if they are given in rough form over here and I manage to get it working successfully.

**Comment by swantzter** [comment]
It seems you just need to set the password and it should just work :tm: https://thelounge.chat/docs/configuration#defaults

**Comment by lynnpepin** [comment]
This bit is unclear to me and I don't see anything in this guide (or the libera.chat SASL guide) that helps explain it. (This is coming from someone coming back to IRC with ~15 years of experience on IRC. I am surprised to find IRC more alienating than Discord!)

Specifically, I am also kicked with the message "SASL authentication to a NickServ account with a verified email address is required to connect from your current network. Please see https://libera.chat/guides/sasl for configuration assistance."

As far as I can tell, this is not a problem with TheLounge, it is intentional behavior from Libera.Chat: It is just impossible to register if you're in a "bad" IP block. (VPN, VPS, unlucky, etc.) You can only register from a "good" IP block. 

I know this is not intended as a help forum, but I hope this can help any wayward souls in search of an answer.

**TLDR:** It appears you must register from a "good" IP first, only clients on "bad" IPs get this message.

**Comment by samebchase** [comment]
Thanks for the details @lynnpepin. Guess we can close this now. I'm able to use TheLounge now.

**Comment by ballerburg9005** [comment]
Libera is totally broken now for lots of people. 

I get this error from dozens of places: when in public wifi, when trying to connect from a friend (cellular, DSL), also from various servers (OVH, Oracle Cloud, AWS, etc.). It makes no sense at all that those IPs are shitlisted, like literally none. With NordVPN I was able to connect simply by switching country. But my home IP didn't work. although I had this exact IP for months. Then I tried from this pure backend server I have access to that has the same IP since over a decade: again the shitlist error. Another server IP: the error. It makes no sense, just broken nonsense. I have access to dozens of servers and VPNs, yeah, me personally I can find some solution. 

But what about all those other people with just normal internet connections, without VPNs, without servers?

It is broken. RIP Libera. No point in even using that 4th or 5th server of choice. 

RIP and double and triple RIP. No network can recover from this.
